test/fixtures/dialects/clickhouse/create_table.yml

```diff
@@ -507,11 +507,10 @@ file:
           naked_identifier: s
           data_type:
             data_type_identifier: Nullable
-            bracketed_arguments:
-              bracketed:
-                start_bracket: (
-                data_type_identifier: String
-                end_bracket: )
+          bracketed:
+            start_bracket: (
+            word: String
+            end_bracket: )
       - end_bracket: )
```

Comment:
This change looks like a regression. While it won't be a `bracketed_arguments` anymore, it should still be part of the `data_type` segment. Can you make sure your change supports `Nullable` types appropriately?

Reply:
oh you're right, I'll look into it

Reply:
I decided to handle the nullable case more explicitely than before